### PR TITLE
Added the CRUD operation for Albums

### PR DESCRIPTION
**Pull request by shrey1608**
Added The following operations:
- User can create an album
- User can delete an album
- User can Update the album 
- CRUD dashboard

### CODE REVIEW

**Review by ExpressHermes — COMMENTED** [review]
Album create template and album update are basically same except the heading and button name(create/update). Use the same template like "`form.html`" For both these functions

**Review by ExpressHermes — COMMENTED** [review]
Rest of the code is good! 👍

**Comment by ExpressHermes** [comment]
The `form.html` template can be used not just for albums but for Songs Or any other model. To update the title of the page dynamically, pass the title as a variable in `context` dict inside the views.

**Comment by shrey1608** [comment]
Done! @ExpressHermes @mann27

**Review by ExpressHermes — APPROVED** [review]

**Comment by shrey1608** [comment]
@mann27 @ExpressHermes  can you give me points after 5-10 days?

**Comment by ExpressHermes** [comment]
I don't understand? You want to delay your points?

**Comment by shrey1608** [comment]
yeah! like its kind of a strategy to win . Only if its possible and if you guys approve it oresle its fine. @ExpressHermes @mann27

**Comment by mann27** [comment]
Is this something psychological to let the others be on the top and then jump at the end, giving them no time to make a comeback 😅
@shrey1608  Let us know when you want your points, no issues!

**Comment by ExpressHermes** [comment]
NIce strategy 😄.

**Comment by ExpressHermes** [comment]
Merging this now!

File: fpage/templates/fpage/albumadd.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                <th scope="col">Action</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for album in all_albums %}
+            <tr>
+                <td>{{ album.album_title }}</td>
+                <td>{{ album.artist }}</td>
+                <td>{{ album.genre }}</td>
+                <td>{{ album.album_logo_link }}</td>
```
Comment (mann27) [inline fpage/templates/fpage/albumadd.html:25]: Do this in a <a/> tag Or remove it there is no need of link when we can't see the image or open it 😅.
( If you can make a popup which will show the image on click that is better but let it be for future issues )

**Comment by shrey1608** [comment]
@mann27  Haha yes that's true and also when people are at top they don't have a goal to reach or to beat someone so they go easy on competing and psychologically they feel at top and don't work at the same levels. Sure i will notify you here when i need points. Thanks a lot guys! @ExpressHermes

**Review by mann27 — COMMENTED** [review]
the better option will be to remove it rn.

File: fpage/templates/fpage/albumadd.html (hunk 1 above)
Comment (mann27) [reply fpage/templates/fpage/albumadd.html:25]: If the link is way too long(can't limit them as they are external) the whole UI will get distorted.

Comment (shrey1608) [reply fpage/templates/fpage/albumadd.html:25]: yes i thought about that too, lets remove it from the crud dashboard but we will have to give an option to upload the image in creat and update views.

Comment (mann27) [reply fpage/templates/fpage/albumadd.html:25]: We can change the link to actual image upload but that would require online cloud storage. Checkout AWS S3 and firebase storage for that.

**Comment by mann27** [comment]
> @mann27 Haha yes that's true and also when people are at top they don't have a goal to reach or to beat someone so they go easy on competing and psychologically they feel at top and don't work at the same levels. Sure i will notify you here when i need points. Thanks a lot guys! @ExpressHermes

@shrey1608 You might wanna redeem your points now. There is going to be a mid-way evaluation before 5th Nov probably.

**Comment by shrey1608** [comment]
> > @mann27 Haha yes that's true and also when people are at top they don't have a goal to reach or to beat someone so they go easy on competing and psychologically they feel at top and don't work at the same levels. Sure i will notify you here when i need points. Thanks a lot guys! @ExpressHermes
> 
> @shrey1608 You might wanna redeem your points now. There is going to be a mid-way evaluation before 5th Nov probably.

yeah can you guys give it to me on 3rd november? As the mid evaluations would be counted till 5th so mine will be counted.Thanks!

**Comment by ExpressHermes** [comment]
Just ping us on 3rd November once

**Comment by shrey1608** [comment]
Sure Done!

File: fpage/templates/fpage/albumadd.html (hunk 1 above)
Comment (shrey1608) [reply fpage/templates/fpage/albumadd.html:25]: I think AWS s3 is paid but we can do in firebase . we will cover it in further issues

**Comment by shrey1608** [comment]
@mann27 please reward me points!

**Comment by mann27** [comment]
give @shrey1608 150 points

**Pull request merged**